### DISCUSSION TITLE
Extra Networks - Description Box UI

### DISCUSSION THREAD

**Discussion by i-liek-katz** [vladmandic/sdnext#1640] [Ideas]
I just updated, and I like how the search is right next to the rest of the ui, good change. Also noticed you can customize the width of the side bar, also good change. As for the description box, this is a negative change for a description of decent length. The forced word wrap could be good, if the box was wider or could be customized to be wider. The real estate is still there to expand the width too, because as of right now, with forced word wrapping, a modest description can make the box pretty long vertically.

**Comment by vladmandic** [comment]
i'll check on that. i just wanted to push incremental improvements as much as possible as this was a painful area for a long time.

**Reply by i-liek-katz** [reply]
I can see now that the current size is probably good for if sidebar is the user's choice of setting, maybe lock it for that, but it is rather constrictive for cover and inline.
